### PR TITLE
bump: minor visual fixups

### LINKED ISSUE #9: Add distinct color for object properties
Add distinct color for object properties

Catppuccin Mocha
Screenshot 2024-03-25 at 19 03 09
Ayu Dark
Screenshot 2024-03-25 at 19 03 41

### PR DESCRIPTION
**Pull request by tecandrew**
1. bolder markdown headers (undocumented)

(before)
image

(after)
image

2. better copilot autocomplete coloring
image

3. set properties (+ ensure [defined Zed syntax](https://github.com/zed-industries/zed/blob/5361a4d72dfe8f286581896e9aa7feb232672493/crates/theme/src/styles/syntax.rs) defaults are Catppuccin colors)

image

fixes #9

### CODE REVIEW

**Comment by tecandrew** [comment]
@sgoudham curious if you think the [default Zed syntax tree-sitter colors](https://github.com/zed-industries/zed/blob/5361a4d72dfe8f286581896e9aa7feb232672493/crates/theme/src/styles/syntax.rs) are OK to be set to default Catpuucin colors instead (as opposed to waiting for the style-guide completion)

File: template.hbr [hunk 1]
```diff
@@ -204,11 +210,21 @@ players:
                     "font_style": null,
                     "font_weight": 700
                 },
+                "enum": {
+                    "color": "#{{maroon}}",
```
Comment (sgoudham) [inline template.hbr:214]:
```suggestion
                    "color": "#{{teal}}",
```

As per VSCode + JetBrains, will update the style guide after this.

image

File: template.hbr [hunk 2]
```diff
@@ -204,11 +210,21 @@ players:
...
+                    "font_style": null,
+                    "font_weight": 700
+                },
                 "function": {
                     "color": "#{{blue}}",
                     "font_style": "italic",
                     "font_weight": null
                 },
+                "hint": {
+                    "color": "#{{maroon}}",
```
Comment (sgoudham) [inline template.hbr:224]: I think `hint` would come under Information? 

image

This is what I'm thinking what a `hint` is:

image

```suggestion
                    "color": "#{{teal}}",
```

**Review by sgoudham — CHANGES_REQUESTED** [review]
Order of priority is probably:

1. Style Guide
2. VSCode
3. Zed or Neovim (Probably favour neovim here since its more familiar to more users)

With 1. being the highest

File: template.hbr (hunk 2 above)
Comment (tecandrew) [reply template.hbr:224]: makes sense to categorize it under Information. i'll make the change

**Pull request merged**